frontend/src/utilities/valueUnits.ts

```diff
@@ -31,7 +31,7 @@ export const splitValueUnit = (
   options: UnitOption[],
 ): [value: number, unit: UnitOption] => {
   const match = value.match(/^(\d*\.?\d*)(.*)$/);
-  if (!match) {
+  if (!(match && match[1])) {
     // Unable to match a legit value -- default back to base
     return [1, options[0]];
   }
```

Comment:
(Line 38) My test it will never get into that case because match[1] will be empty if value does not start with numbers. Unsure how to proceed with writing a test or fixing this IF statement.